incubator/mariadb-cluster/templates/slave-statefulset.yaml

```diff
@@ -41,16 +45,10 @@ spec:
                   app: "{{ template "name" . }}"
                   release: "{{ .Release.Name }}"
       {{- end }}
-      {{- if .Values.image.pullSecrets }}
-      imagePullSecrets:
-      {{- range .Values.image.pullSecrets }}
-        - name: {{ . }}
-      {{- end}}
-      {{- end }}
       containers:
       - name: "mariadb"
-        image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-        imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+        image: "{{ .Values.image }}"
+        imagePullPolicy: "{{ .Values.pullPolicy }}"
```

Comment:
I believe you should revert this change. We're now following the following structure for specifying the chart image

```
image:
  registry: docker.io
  repository: bitnami/mariadb
  tag: 1.2.3
  pullPolicy: Always
```

The elasticsearch chart PR by @juan131 adds a nifty helper for this substitution. It's worth re-using it https://github.com/juan131/charts-1/blob/a9cfe16ed3f5f678fb9b1bae138b5f75739c7f97/incubator/elasticsearch/templates/_helpers.tpl#L25-L32

Reply:
@beltran-rubo I think you miss this comment